src/iframe-transport-common.js

```diff
@@ -19,9 +19,16 @@
  * This is the type of message that will be sent to the 3p frame.
  * The message will contain an array of the typedef declared below.
  */
-export const IFRAME_TRANSPORT_EVENTS_TYPE = 'IframeTransportEvents';
+export const IFRAME_TRANSPORT_EVENTS = 'iframe-transport-events';
```

Comment:
actually, it probably makes sense to merge this whole file into `3p-frame-messaging.js` now